scripts/generate_thumbnail.py

```diff
@@ -32,12 +32,16 @@
 with open(data_path, 'rt', encoding='utf-8') as f:
     data = json.loads(f.read())
 
+print('nopeeee')
+
 game_codename = data.get("game").get("codename")
-asset_data_path = f"../user_data/games/{game_codename}/{used_assets}/config.json"
+asset_data_path = f"./user_data/games/{game_codename}/{used_assets}/config.json"
```

Comment:
Not a big deal given that there's a button in the UI now, but that change will probably break the BAT file

Reply:
Ah yes, I shouldn't commit changes on `./scripts/generate_thumbnail.py` as I copied and moved it inside `./src/thumbnail/`